exercises/say/say_test.py

```diff
@@ -2,8 +2,7 @@
 
 from say import say
 
-
```

Comment:
Please restore this blank line.

The line spacing around the canonical data reference comment is 2 above, 1 below.